src/coreclr/src/gc/gc.cpp

```diff
@@ -30262,6 +30291,9 @@ gc_heap::mark_through_cards_helper (uint8_t** poo, size_t& n_gen,
     int thread = hpt->heap_number;
 #else
     THREAD_FROM_HEAP;
+#if defined (MULTIPLE_HEAPS)
+    gc_heap* hpt = this;
```

Comment:
This is just to fix build if someone wants to disable `FEATURE_CARD_MARKING_STEALING`. 
It could be useful to disable that when investigating races.